src/shared/components/Glider/Glider.tsx

```diff
@@ -86,7 +86,7 @@ export function useGlider<T extends HTMLElement>({
 function useEventListener<K extends keyof GliderEventMap>(
   element: HTMLElement | undefined | null,
   event: K,
-  listener: (event: GliderEvent<GliderEventMap[K]>) => void = () => {}
+  listener: (event: GliderEvent<GliderEventMap[K]>) => void = () => ({})
```

Comment:
Why was this modified? I don't think Glider is used in any way here?

Reply:
I noticed this line was producing an annoying lint warning for every PR as empty arrow functions are not allowed

Reply:
I see. This however changes the signature of the function and now it will always return a truthy `{}` value, and I'm not sure if this does or doesn't have any implications. I suggest to skip it here as it's not part of the scope. Post-release we should do a general cleanup of all the eslint warning and we can take care of it then